lib/views/list_view_page.dart

```diff
@@ -1,91 +1,96 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:todo_app/cubit/todolist_cubit.dart';
 import 'package:todo_app/views/detailed_todo_page.dart';
 
 import '../models/todo.dart';
 
-class ListViewPage extends StatefulWidget {
+class ListViewPage extends StatelessWidget {
   const ListViewPage({Key? key}) : super(key: key);
-
-  @override
-  _ListViewPageState createState() => _ListViewPageState();
-}
-
-class _ListViewPageState extends State<ListViewPage> {
-  final _todoList = <Todo>[
-    Todo(title: 'Two-line item', description: 'description'),
-    Todo(title: 'Two-line item', description: 'description'),
-    Todo(title: 'Two-line item', description: 'description'),
-  ];
-
-  void handleCheckbox(bool? newValue, int index) => setState(
-        () {
-          _todoList[index].done = newValue!;
-        },
-      );
-
   @override
   Widget build(BuildContext context) {
-    return Container(
-      color: Colors.white,
-      child: ListView.separated(
-        itemBuilder: ((context, index) {
-          if (index == _todoList.length) return const ClearDoneTodos();
+    final cubit = BlocProvider.of<TodolistCubit>(context);
+
+    return BlocBuilder<TodolistCubit, List<Todo>>(
+      builder: (context, state) {
+        return Container(
+          color: Colors.white,
+          child: state.isNotEmpty
+              ? ListView.separated(
```

Comment:
i think it would be good to put this widgets into functions so the conditional its show better